.github/actions/oblt-cli-create-serverless/action.yml

```diff
@@ -37,71 +37,50 @@ inputs:
-          echo -n '"GitHubRepository":"${{ github.repository }}"'
-          [ -n '${{ github.event.issue.number }}' ] && echo -n ',"GitHubIssue":"${{ github.event.issue.number }}"'
-          echo -n '}'
-        } > .gitops
-        echo "GITOPS=$(cat .gitops)" >> $GITHUB_ENV
-        rm .gitops
-      shell: bash
-
-    - name: Configure oblt-cli flags for gitops - pull-request
-      if: github.event.issue.pull_request && contains(inputs.gitops, 'true')
-      run: |
-        {
-          echo -n '{'
-          echo -n '"GitOps":"true",'
-          echo -n '"GitHubRepository":"${{ github.repository }}",'
-          echo -n '"GitHubCommit":"${{ github.event.comment.id }}",'
-          echo -n '"GitHubPullRequest":"${{ github.event.issue.number }}"'
-          echo -n '}'
-        } > .gitops
-        echo "GITOPS=$(cat .gitops)" >> $GITHUB_ENV
-        rm .gitops
-      shell: bash
-
-    - name: Configure oblt-cli serverless parameters
-      run: |
-        {
-          echo -n '{'
-          echo -n '"Target":"${{ inputs.target }}",'
-          echo -n '"ProjectType":"${{ inputs.project-type }}"'
-          echo -n '}'
-        } > .serverless
-        echo "SERVERLESS=$(cat .serverless)" >> $GITHUB_ENV
-        rm .serverless
-      shell: bash
+      uses: actions/github-script@v6
+      with:
+        script: |
+          console.log(`Parsing flags`)
+          const { cluster_name_prefix, cluster_name_suffix, dry_run, gitops, target, project_type  } = process.env
+          if (cluster_name_prefix != '') {
+            core.exportVariable('CLUSTER_NAME_PREFIX', `--cluster-name-prefix=${cluster_name_prefix}`)
+          }
+          if (cluster_name_suffix != '') {
+            core.exportVariable('CLUSTER_NAME_SUFFIX', `--cluster-name-suffix=${cluster_name_suffix}`)
+          }
+          if (dry_run != 'false') {
+            core.exportVariable('DRY_RUN', `--dry-run`)
+          }
+          var parameters = {}
+          parameters.ProjectType = project_type
+          parameters.Target = target
 
-    - name: Configure oblt-cli merge parameters
-      run: |
-        PARAMETERS=$(echo '${{ env.SERVERLESS }} ${{ env.GITOPS }}' | jq -c -s add)
-        echo "PARAMETERS=${PARAMETERS}" >> $GITHUB_ENV
-      shell: bash
+          if (gitops == 'true') {
+            console.log(`Parsing gitops flags`)
+            parameters.GitOps = "true"
+            parameters.GitHubRepository = `${context.repo.owner}/${context.repo.repo}`
+            if (context.eventName == 'issues') {
+              if (context.issue.number) {
+                parameters.GitHubIssue = context.issue.number
+              }
+            } else {
+              parameters.GitHubPullRequest = context.issue.number
+            }
+          }
+          console.log(`Create PARAMETERS env variable`)
+          core.exportVariable('PARAMETERS', `--parameters='${(JSON.stringify(parameters))}'`)
+      env:
+        cluster_name_prefix: ${{ inputs.cluster-name-prefix }}
```

Comment:
I could not use `process.env` since those variables contain some `hyphens` and somehow I faced some errors. So I decided to use the env approach to solve my initial problem and also to protect from any input injections, as explained in https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable